_data/projects/GenAI-API.yml

```diff
@@ -0,0 +1,17 @@
+name: genai-api
+desc: Supercharge Apple’s Shortcuts using Cloudflare Workers and Gemini within minutes (and for free, up to 1,500 requests per day) ☁️✨
+site: https://github.com/louisbrulenaudet/genai-api
+tags:
+  - api
+  - cloudflare
+  - workers
+  - shortcuts
+  - apple
+  - hono
+  - gemini
+  - genai
+  - ai
+  - llm
+upforgrabs:
+  name: Good First Issue
+  link: https://github.com/louisbrulenaudet/genai-api/issues
```

Comment:
This link should point to a label under the repository, but the label on the previous line doesn't exist in https://github.com/louisbrulenaudet/genai-api/labels and there are no open issues